### DISCUSSION TITLE
Plans and Roadmap

### DISCUSSION THREAD

**Discussion by jfreak53** [eduardogsilva/routerfleet#25] [General]
So I love this project, it's been on my radar for awhile, and if something didn't come up I had planned on starting a similar project for my own needs.

Currently, the system only does backups, which is great. However, what is your roadmap for this, plans for the future? I know ROS has a lot of features, a lot, so implementing all features would be a task in and of itself. Do you plan on implementing the ability to change settings in the routers we are connected to at all?

Thanks!

**Comment by eduardogsilva** [comment]
Hello @jfreak53 

The current plans are to keep all compatible equipment with the same feature set. So if I implement a "reboot" function, it must (or at least should) work on all routers.

Routerfleet is not intended to replace the equipment's management interface. I plan to add some functions to help on admin tasks, or maybe to allow to run or deploy scripts, but currently, they aren't mapped yet.

I'm still working on the foundations of routerfleet.

Cheers!!
